activefires_pp/tests/test_spatiotemporal_alarm_filtering.py

```diff
@@ -529,16 +553,19 @@ def test_alarm_filter_runner_call_spatio_temporal_alarm_filtering_has_alarms(sen
     assert result[0] == alarm
 
 
-@patch('activefires_pp.spatiotemporal_alarm_filtering.get_xauth_environment_variable')
+@patch('activefires_pp.spatiotemporal_alarm_filtering._get_xauthentication_token')
+@patch('activefires_pp.spatiotemporal_alarm_filtering.get_xauthentication_filepath_from_environment')
```

Comment:
this is being repeated three times. Would a fixture with an actual (temporary) file containing the xauth be a solution?

Reply:
This should be better now!